run--01a--prep-geo-data.Rmd

```diff
@@ -533,17 +664,101 @@ if (exists("my_aux_geo")) {
              County = case_when(max(in_aux_flag) == 0 ~ County,
                                 in_aux_flag      == 0 ~ paste(County, glue("(Non-{my_aux_overall_name})")),
                                 in_aux_flag      == 1 ~ paste(County, glue("({my_aux_overall_name})"))),
-             pct_area = ifelse(is.na(pct_area), 1, pct_area))
+             pct_area = ifelse(is.na(pct_area), 1, pct_area)) %>% 
+      mutate(County = ifelse(County %like% "DuPage", "DuPage", County))
     
     if (FALSE) {
       cbind(
         geo_crosswalk     %>% filter(chicago_flag == 1) %>% select(GEOID, pct_area) %>% arrange(GEOID), 
         geo_crosswalk_aux %>%                               select(GEOID, pct_area) %>% arrange(GEOID))
     }
   }
+} else {
+  geo_crosswalk <- geo_crosswalk %>% 
+    mutate(pct_area = 1)
 }
 ```
 
+```{r add Birth to Five Council for Illinois}
+if (my_state_abbr == "IL") {
+  b5_sd_file <- glue("{input_path}BirthToFive_SD2023_lookup.xlsx")
+  b5_sd <-
+    readxl::read_xlsx(b5_sd_file) %>% 
+    rename(school = SchoolDistrict_NAME_Census,
+           b5     = BirthToFive_regionlabel)
+  
+  geo_crosswalk_sd <- geo_crosswalk_sd %>% 
+    mutate(school = case_when(school == "Chester-East Lincoln Community CSD 61" ~ "Chester-East Lincoln Community Csd 61",
+                              TRUE                                              ~ school))
+  
+  geo_crosswalk_b5 <- geo_crosswalk_sd %>% 
+    merge(b5_sd %>% as.data.frame() %>% select(school, b5) %>% unique(),
+          by = "school",
+          all.x = TRUE) %>% 
+    group_by(GEOID) %>% 
+    mutate(pct_b5 = LANDAREA / sum(LANDAREA)) %>% 
```

Comment:
Not critical but: it's better practice to add an argument of `.by = GEOID` inside of the `mutate()`, as it avoids potential issues with the resulting tibble being grouped, which can be unpredictable.